### DISCUSSION TITLE
Allow Users  to Only See the Terminal Screen

### DISCUSSION THREAD

**Discussion by TikursewMe** [cockpit-project/cockpit#19185] [Q&A] [answered]
how do i configure cockpit so that it only allows users to see the terminal screen page and disable any other screens on the page?

**Comment by martinpitt** [comment, marked as answer]
Either with [manifest overrides](https://cockpit-project.org/guide/latest/packages.html#package-manifest-override), or you remove all other cockpit pages in /usr/share/cockpit/, or with [embedding](https://cockpit-project.org/guide/latest/embedding.html). The latter shows that you can also only show the "Terminal" frame, without any shell around it.
